src/edge/ssr/__init__.py

```diff
@@ -0,0 +1,601 @@
+from Bio.Seq import Seq
+
+
+# sites separated by more than these number of bps will not be considered as
+# part of the same combination
+SITE_SEPARATION_MAX = 50000
+
+
+def rc(s):
+    return str(Seq(s).reverse_complement())
+
+
+def _c(s):
```

Comment:
would be nice to to have a non-single letter name for this like "normalized_sequence" or something